pvlib/test/test_solarposition.py

```diff
@@ -45,6 +47,22 @@ def expected_solpos_multi():
                          'apparent_elevation': [39.888378, 39.521740]},
                         index=[['2003-10-17T12:30:30Z', '2003-10-18T12:30:30Z']])
 
+
+@pytest.fixture()
+def expected_rise_set():
+    # for Golden, CO, from USNO website
+    times = pd.DatetimeIndex([datetime.datetime(2015, 1, 2),
+                              datetime.datetime(2015, 8, 2),
```

Comment:
We should also test times `sunrise < time < sunset` and `sunset < time < midnight`. Probably also document the expected behavior in the function docstring.

Reply:
Sorry, didn't realize this was factored out of the test function below. I guess that complicates things. Up to you on how to proceed.

Reply:
Absolutely.  Once I get this working I'll extend the test conditions, where we supply times before and after sunrise and sunset, times before and after midnight UTC, etc.